src/coreComponents/physicsSolvers/surfaceGeneration/SurfaceGenerator.cpp

```diff
@@ -522,9 +523,9 @@ int SurfaceGenerator::separationDriver( DomainPartition & domain,
   FieldIdentifiers fieldsToBeSync;
 
   fieldsToBeSync.addFields( FieldLocation::Face, { extrinsicMeshData::RuptureState::key() } );
-  if( nodeManager.hasWrapper( SolidMechanicsLagrangianFEM::viewKeyStruct::forceExternalString() ) )
+  if( nodeManager.hasWrapper( extrinsicMeshData::solidMechanics::externalForce::key() ) )
```

Comment:
I think we have a `hasExtrinsicData` for this kind of pattern?

Reply:
Done, I replaced all the `hasWrapper` with `hasExtrinsicData`.